sdk/python/kfp/client/client.py

```diff
@@ -398,9 +398,14 @@ def set_user_namespace(self, namespace: str) -> None:
         with open(Client._LOCAL_KFP_CONTEXT, 'w') as f:
             json.dump(self._context_setting, f)
 
-    def get_kfp_healthz(self) -> kfp_server_api.ApiGetHealthzResponse:
+    def get_kfp_healthz(
+            self,
+            sleep_duration: int = 5) -> kfp_server_api.ApiGetHealthzResponse:
```

Comment:
Without this parameter, this small test takes 15 seconds to execute due to the 5 second default. We default it to 5 so there is no change to end users from a UX/API standpoint.